src/create-express-middleware.js

```diff
@@ -6,7 +6,7 @@ import { mergeFeatureNames } from './merge-feature-names';
 import { getQueryFeatureNames } from './get-query-feature-names';
 
 const setStatus = (res, isActiveFeatureName) =>
-  isActiveFeatureName ? res.status(200) : res.status(404);
+  isActiveFeatureName ? res.status(200) : res.status(404).send();
```

Comment:
Should we set the status here at all if the feature is enabled?

Reply:
If this is the only place the status gets set...

Reply:
What do you think?

Reply:
Maybe the line that calls this should be:

```
if (!isActiveFeatureName(requiredFeature, updatedFeatures) {
  return res.status(404).send();
}
```

Don't call the handler. Don't call `next()`.